src/server/scripts/Northrend/Naxxramas/boss_thaddius.cpp

```diff
@@ -110,168 +155,270 @@ class boss_thaddius : public CreatureScript
         return GetInstanceAI<boss_thaddiusAI>(creature);
     }
 
-    struct boss_thaddiusAI : public BossAI
+    class boss_thaddiusAI : public BossAI
```

Comment:
well, you can have stuff as private even with struct, and we are used to use structs there ;)